tests/integration/clientv3/metrics_test.go

```diff
@@ -177,3 +181,189 @@ func getHTTPBodyAsLines(t *testing.T, url string) []string {
 	resp.Body.Close()
 	return lines
 }
+
+func TestAllMetricsGenerated(t *testing.T) {
```

Comment:
The test is large already, can you move the test to separate PR?

Reply:
of course @serathius

Reply:
Done